addons/hr_expense/models/hr_expense.py

```diff
@@ -96,6 +100,8 @@ def _get_employee_id_domain(self):
         ('done', 'Paid'),
         ('refused', 'Refused')
     ], compute='_compute_state', string='Status', copy=False, index=True, readonly=True, store=True, default='draft', help="Status of the expense.")
+    # We want to see the records that are just processed by OCR at the top of the list.
+    state_processed = fields.Boolean(string='Status regarding OCR status', default=False)
```

Comment:
The OCR is not used in hr_expense. To respect our modular approach, this should be defined in ``hr_expense_extract``

Reply:
Indeed, I will move it :)